frontend/src/views/Org/MembersPage/components/OrgIdentityTab/components/IdentitySection/IdentityTable.tsx

```diff
@@ -36,22 +48,58 @@ type Props = {
   ) => void;
 };
 
+const INIT_PER_PAGE = 10;
+
 export const IdentityTable = ({ handlePopUpOpen }: Props) => {
   const router = useRouter();
   const { currentOrg } = useOrganization();
-  const orgId = currentOrg?.id || "";
+  const [page, setPage] = useState(1);
+  const [perPage, setPerPage] = useState(INIT_PER_PAGE);
+  const [direction, setDirection] = useState("asc");
```

Comment:
Enums would be better option here as state

Reply:
you're referring to the order direction and order by column correct? not the page/per page count?

Reply:
Yes